tasks/event_tasks.py

```diff
@@ -57,7 +59,8 @@ def session_event_task_factory(task_params=None):
     session_data_tmpl = (
         '{{"sent_at":"{started}"}}\n'
         + '{{"type":"session"}}\n'
-        + '{{"init":true,"started":"{started}","status":"exited","errors":0,"duration":0,"attrs":{{"release":"{release}"}}}}'
+        + '{{"init":true,"started":"{started}","status":"exited","errors":0,"duration":0,"attrs":{{"release":"{'
```

Comment:
_nit_: Can we join this into one line to improve readability?

Reply:
ok... disabling line wrapping on reformat.